src/test/kotlin/org/rustSlowTests/lang/resolve/CargoProjectResolveTest.kt

```diff
@@ -141,6 +142,60 @@ class CargoProjectResolveTest : RsWithToolchainTestBase() {
         checkReferenceIsResolved<RsPath>("src/bar.rs")
     }
 
+    fun `test resolve local package under symlink`() = fileTree {
```

Comment:
Ideally we also need test when project directory is under symlink, but I couldn't get it work yet:
```kotlin
    fun `test module relations when project is under symlink`() = fileTree {
        toml("Cargo.toml", """
            [package]
            name = "mods"
            version = "0.1.0"
            authors = []

            [dependencies]
        """)

        dir("src") {
            rust("lib.rs", """
                mod foo;

                pub struct S;
            """)

            rust("foo.rs", """
                use S;
                  //^
            """)
        }
    }.let { fileTree ->
        if (SystemInfo.isWindows) return@let
        val rootPath = cargoProjectDirectory.pathAsPath
        val targetName = cargoProjectDirectory.name + "_target"
        val targetPath = rootPath.parent.resolve(targetName)
        Files.move(rootPath, targetPath)
        Files.createSymbolicLink(rootPath, targetPath)

        fullyRefreshDirectoryInUnitTests(cargoProjectDirectory.parent)
        val target = cargoProjectDirectory.parent.findChild(targetName)!!
        val project = fileTree.create(project, target)
        refreshWorkspace()
        project.checkReferenceIsResolved<RsPath>("src/foo.rs")
    }
```

Reply:
I got test working locally by changing
```kotlin
val project = fileTree.create(project, target)
```
to
```kotlin
val project = fileTree.create(project, cargoProjectDirectory)
```
but unfortunately test still doesn't pass on CI